src/oc/notify/async/user.clj

```diff
@@ -4,13 +4,14 @@
 
   Use of this is through core/async. A message is sent to the `user-chan`.
   "
-  (:require [clojure.core.async :as async :refer (>!! <!)]
-            [defun.core :refer (defun-)]
-            [taoensso.timbre :as timbre]
-            [oc.lib.db.pool :as pool]
+  (:require [clojure.core.async :as async :refer [<! >!!]]
+            [defun.core :refer [defun-]]
```

Comment:
Not sure if there is a specific reason why you changed these but we usually prefer round brackets for readability.
No need to change, just for the next time.

Sean made me notice it and i agree it's better.